packages/style-engine/class-wp-style-engine-processor.php

```diff
@@ -12,26 +12,52 @@
 }
 
 /**
- * Compiles styles from a store of CSS rules.
+ * Compiles styles from stores or collection of CSS rules.
  *
  * @access private
  */
 class WP_Style_Engine_Processor {
 
 	/**
-	 * The Style-Engine Store object.
+	 * The Style-Engine Store objects
 	 *
-	 * @var WP_Style_Engine_CSS_Rules_Store
+	 * @var WP_Style_Engine_CSS_Rules_Store[]
 	 */
-	protected $store;
+	protected $stores = array();
 
 	/**
-	 * Constructor.
+	 * The set of CSS rules that this processor will work on.
 	 *
-	 * @param WP_Style_Engine_CSS_Rules_Store $store The store to render.
+	 * @var WP_Style_Engine_CSS_Rule[]
 	 */
-	public function __construct( WP_Style_Engine_CSS_Rules_Store $store ) {
-		$this->store = $store;
+	protected $css_rules = array();
+
+	/**
+	 * Add a store to the processor.
+	 *
+	 * @param WP_Style_Engine_CSS_Rules_Store $store The store to add.
+	 */
+	public function add_store( WP_Style_Engine_CSS_Rules_Store $store ) {
+		$this->stores[] = $store;
```

Comment:
What happens if we call `add_store` twice with the same store? Should the stores be named, as they are in `WP_Style_Engine_CSS_Rules_Store` so we avoid duplication? Or should the Processor not really care about knowing anything about the stores 🤔

Reply:
> Or should the Processor not really care about knowing anything about the stores 

I would guess "yes" to this, and I suppose it's less of a risk since we instantiate `WP_Style_Engine_Processor` when we want to use it. 

I suppose it wouldn't be much of a change to allow for keys though. 

Interested to hear @aristath's take on it.

Reply:
I'm happy either way with this one, just thought I'd ask 🙂

Reply:
You're right, it would make sense to have them named here. I pushed a change and now there won't be duplicates 👍